practice-one/src/components/CustomerAvatar/CustomerAvatar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./CustomerAvatar.css";
+
+const CustomerAvatar = ({ customerAvatar, customerName }) => {
+  return (
+    <div className="customer__wrapper">
+      <div className="customer__avatar">
+        <img
+          className="customer__img"
+          src={customerAvatar}
```

Comment:
avatar and name is fine

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/7bf0e11b-ab6f-4ef2-86d7-7f6a62b85f64)
![image](https://github.com/datdragon777/React-Training/assets/74806826/5369e910-cdd6-4ec5-869e-5f6dce4207d2)